working-with-your-data/working-with-your-own-data.md

```diff
@@ -114,6 +119,84 @@ wget --user=<USERNAME> --ask-password <URL>
+
+Click the big green `Download` button.
+
+Click `Download` on this next page for `FileZilla` this is the only free option but will have the functionality you need.
+
+<img src="screenshots/filezilla-download.png">
+
+After download is complete, you'll find the `FileZilla`'s `.app.tar.bz2` file in your download files or you can click on it in the corner of your web browser's screen.
+Double click on the file to install.
+
+You may want to move the App file to where your other applications are stored.
+
+#### Install FileZilla on Windows
+
+Go to [FileZilla's website](https://filezilla-project.org/download.php?type=client) to download the FileZilla Client.
+
+Click the big green `Download` button.
+
+Click `Download` on this next page for `FileZilla` this is the only free option but will have the functionality you need.
+
+<img src="screenshots/filezilla-download.png">
+
+After download is complete, you'll find the `FileZilla` `.exe` file in your download files or you can click on it in the corner of your web browser's screen.
+
+Double click on the file to install to begin installation.
+
+You'll be asked if you want to `Allow FileZilla to make changes` click `Yes`.  
+
+There will be a series of steps (like below) you need to click `Next` and `Accept` to them.
+
+<img src="screenshots/filezilla-windows.png" width=600>
+
+#### Linking FileZilla to the RStudio Server
+
+Open up the FileZilla application.
+At the top of the FileZilla screen, you can enter in the address and your credentials for our RStudio Server (send a message to one of our staff if you forgot your username or password).
+
+<img src="screenshots/filezilla-bar.png">
+
+
+For `Host`, type in `rstudio.ccdatalab.org`.  
+For `Username`, type in the username you use to login in to our RStudio server.  
+For `Password` type in the password you use to login in to our RStudio server.  
+For `Port`, type in `22`.  
+
+
+Then click the blue `Quickconnect` button.
+FileZilla may ask you if you want it to remember your passwords.
+We'd suggest creating a master password or using `Do not save password`.
+
+<img src="screenshots/filezilla-password.png" width=500>
+
+Next, FileZilla will ask you if you should trust our RStudio Server.
+You can check the box for `Always trust this host` if you don't want to be asked this again.
+Then click `OK`.
+
+<img src="screenshots/filezilla-trust-server.png" width=500>
+
+#### Using FileZilla to upload files to the RStudio Server
+
+The left side of the FileZilla screen is the file/folder you'd like to upload and the right side is where in the RStudio Server you'd like to upload to.
+
+On the right side, click on the folder you'd like to upload the files to in the RStudio Server.
+
+Then, on the left, navigate to the file or folder on your computer you'd like to upload the RStudio Server.
+You may be asked to allow FileZilla to have access to your files.
+Click `OK` for each time.
+
+<img src="screenshots/filezilla-permission.png" width=400>
```

Comment:
Would you consider this part Mac-specific?

Reply:
From my experimenting it seemed it was.

Reply:
Can we write “on a Mac” in that case? I did not have any permissions issue/step on Ubuntu.